packages/nitro-protocol/contracts/ninja-nitro/AdjudicatorFactory.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.4;
+pragma experimental ABIEncoderV2;
+
+import '@openzeppelin/contracts/utils/Create2.sol';
+import './SingleChannelAdjudicator.sol';
+
+contract AdjudicatorFactory {
```

Comment:
It might be nice to use the openzeppelin [`clones` library f](https://docs.openzeppelin.com/contracts/3.x/api/proxy#Clones)or this.